.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+node_js: node
+cache: yarn
+
+install:
+  - yarn install
+
+script:
+  - yarn test
+  - yarn run build
+
+deploy:
+  provider: pages       
```

Comment:
you are currently deploying on github pages and we need to deploy on firebase

Reply:
@samarv I've changed the configuration to deploy on firebase. Please could you check again. Thanks.